src/baldr/streetname_us.cc

```diff
@@ -1,7 +1,5 @@
-#include <iostream>
-
-#include "baldr/streetname.h"
 #include "baldr/streetname_us.h"
+#include "baldr/streetname.h"
```

Comment:
ill be shocked if the linter doesnt complain about reordering these (i think its alphabetically sorts them)

Reply:
strange, i guess it doesnt care about sorting then!

Reply:
I believe `clang-format` has a specific rule for the header that has the same name as the current source file.